tests/test_hsm.py

```diff
@@ -781,6 +781,30 @@ def test_bounds_centroid():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_ksb_sig():
```

Comment:
Good idea.  I added your suggested test to the testing suite (just for `ksb_sig_factor`).  I think this plus what I had before then covers both `ksb_sig_factor` and `ksb_sig_weight`.